generic_tracers/generic_COBALT.F90

```diff
@@ -870,150 +861,347 @@ subroutine user_add_params(param_file)
-                   default=0.25, scale = micromol2mol / sperd)  ! s-1 (mole N kg)-1
-    call get_param(param_file, "generic_COBALT", "ktemp_vir", cobalt%vir_ktemp,  "ktemp_vir", units="C-1", default= 0.063)           ! C-1
-    !
-    !-----------------------------------------------------------------------
-    ! Phytoplankton losses to mortality
-    !-----------------------------------------------------------------------
+    ! The parameters that follow pertain to a range of different phytoplankton mortality processes other than being
+    ! consumed by zooplankton.  They include aggregation into sinking detrital particles, virus-driven losses (that
+    ! are also applied to bacteria), cell death (i.e., true phytoplankton mortality) and direct phytoplankton sinking. 
+    ! This section also includes parameters for the exudation of fixed carbon by phytoplankton.
+    ! 
+    ! Phytoplankton mortality terms differ by whether they are density-dependent (i.e., linear versus quadratic),
+    ! temperature-dependent, and whether they vary depending on the condition (or stress) of the phytoplankton. The
+    ! level of stress is defined based on the achieved growth rate relative to the maximum growth rate.  By default,
+    ! stress-dependent mortality begins to increase from 0 when this ratio falls below 0.25 (frac_mu_stress = 0.25).
+    !
+    call get_param(param_file, "generic_COBALT", "frac_mu_stress_Sm", phyto(SMALL)%frac_mu_stress, &
+                   "fraction of max growth when stress-dependent losses initiate for small phytoplankton", &
+         units="none", default=0.25)
+    call get_param(param_file, "generic_COBALT", "frac_mu_stress_Di", phyto(DIAZO)%frac_mu_stress, &
+                  "fraction of max growth when stress-dependent losses initiate for diazotrophs", &
+                  units="none", default=0.25)
+    call get_param(param_file, "generic_COBALT", "frac_mu_stress_Lg", phyto(LARGE)%frac_mu_stress, &
+                   "fraction of max growth when stress-dependent losses initiate for large phytoplankton", &
+                   units="none", default=0.25)
+    call get_param(param_file, "generic_COBALT", "frac_mu_stress_Md", phyto(MEDIUM)%frac_mu_stress, &
+                   "fraction of max growth when stress-dependent losses initiate for medium phytoplankton", &
+                   units="none", default=0.25)
+    !
+    ! Phytoplankton aggregation is a density-dependent (quadratic) mortality term that is more effective for large
+    ! phytoplankton, following Jackson (1990).  Aggregation rates increase with stress (e.g., Waite et al, 1992).
+    ! As a primarily physical process of greatest importance in cold ecosystems, aggregation is assumed to be
+    ! temperature-independent.  Rates were informed by Jackson's experiments and tuning to satellite-observed blooms.
+    ! Note: Values for these parameters are entered in day-1 (micromol kg-1)-1 and converted to sec-1 (mol kg-1)-1
```

Comment:
@charliestock , the unit in your comment `day-1 (micromol kg-1)-1` is not consistent with what you put later in `get_param` call (i.e. https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/ocean_BGC/blob/2aa27adc80da33051cce0dc81f6b688db4d91cd0/generic_tracers/generic_COBALT.F90#L935). Can you double-check that and confirm which is the correct unit?